### ISSUE TITLE
Why request access to all URLs?

### ISSUE THREAD

**Issue by Nantris** [Konnor88/anti-grabify#6]
Is there a limit to the number of URLs you can include in the manifest?

**Comment by Konnor88** [comment]
Hi, the reason I decided to use all URLs is that whenever I added new URLs before and pushed out an update, it would disable the extension for the users until they acknowledge the permission changes. This change just makes upgrading it a bit easier.

**Comment by Slapbox** [comment]
That makes sense, and thanks for your reply and your great work on this plugin.

But this change has made me more wary of the plugin and I've left it disabled since it updated, so a bit counterproductive maybe. I previously was happy to enable it without needing to look over the source code since it explicitly limited the sites it allowed itself to.

My two cents, I think it would be better to go back to the old way of explicitly listing sites.

**Comment by Konnor88** [comment]
Yeah, I can see where you're coming from on that. I'll keep that in consideration and will do that if there are more requests to go back to the old way. Thanks for your feedback.

**Comment by boobthecat** [comment]
Bump :B

**Comment by Slapbox** [comment]
I'd still be interested in a version that didn't request access to all URLs.

**Comment by Realswitzer** [comment]
> I'd still be interested in a version that didn't request access to all URLs.

I made a fork to change that, only problem is it being unsigned and unavailable on add-on stores (AMO and CWS primarily). The problem of it being disabled on every update will still be present, along with a lack of auto-updating.
Any updates upstream, if I continue to maintain the fork, would likely end up being delayed too.
https://github.com/Realswitzer/anti-grabify/releases/tag/1.14

**Comment by boobthecat** [comment]
You just forgot to add an ID to the Firefox one. I can't install it, only as temporary in the about:debugging

**Comment by Realswitzer** [comment]
Oh, didn't know that. Will look into and should be able to push a release today.

**Comment by boobthecat** [comment]
Thanks!
